app/assets/stylesheets/list.scss

```diff
@@ -1,3 +1,5 @@
+@import "variables";
```

Comment:
I originally started with this `import` statement, which I think is fine _but_ I wonder if there's a better way we could organize our SASS.

Nothing to change now..just a thought 🤔